credentials/credentials.go

```diff
@@ -30,17 +30,25 @@ type Credentials struct {
 	WorkspaceID  string
 	ClientID     string
 	ClientSecret string
+	BootstrapKey []byte
 }
 
 func New(clientID, clientSecret, privateKey string) (*Credentials, error) {
 	if clientID == "" || clientSecret == "" {
 		return nil, ErrInvalidCredentials
 	}
 
+	h := sha256.New()
+	h.Write([]byte(clientID))
+	h.Write([]byte(":"))
```

Comment:
I don't think we use a `:` separator in the existing code (and we don't really need one, do we?)